Meter.c

```diff
@@ -325,6 +323,13 @@ static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
       data->values[nValues - 1] = sumPositiveValues(this->values, this->curItems);
    }
 
+   if (w <= 0)
+      return;
+   if ((size_t)w > nValues / 2) {
+      x += w - nValues / 2;
+      w = nValues / 2;
+   }
+
```

Comment:
Just some more space for the code to breathe … ;-)

```suggestion
   if (w <= 0)
      return;

   if ((size_t)w > nValues / 2) {
      x += w - nValues / 2;
      w = nValues / 2;
   }

```